### PR TITLE
Fixing fibonacci function

### PR DESCRIPTION
**Pull request by olgaborodina**
Fixed fibonacci function in `simple_functions.py`:

1. Added test for `int` type
2. Added test for minimal value
3. Included the maximum value in the list if `max` is one of the fibonacci numbers

### CODE REVIEW

**Comment by KathrynJones1** [comment]
I would like to review this!

File: participants/Borodina/pr_tutorial/simple_functions.py [hunk 1]
```diff
@@ -1,6 +1,10 @@
 def fibonacci(max):
+    if type(max) != int:
+        raise ValueError('max value should be int')
```
Comment (KathrynJones1) [inline participants/Borodina/pr_tutorial/simple_functions.py]: Maybe spell out integer in full (sorry I just wanted to try out changing something xD).
```suggestion
        raise ValueError('max value should be integer')
```

**Review by KathrynJones1 — COMMENTED** [review]
Looks good :)

**Comment by ceb8** [comment]
Thanks! 😄

**Pull request merged**